datastore/makefile

```diff
@@ -19,11 +19,6 @@ build:
 	cd elastic && go build
 	go build
 
-test: build
-	cd elastic && go test $(GOTEST_FLAGS)
-	cd integration_test && go test $(GOTEST_FLAGS)
-	go test $(GOTEST_FLAGS)
-
```

Comment:
Remove recursive make testing scheme in favor of go test ./... recursive testing scheme.